onnx/backend/test/case/node/index_put.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
```

Comment:
Please also rename this file.

Reply:
The file is already renamed, this code is outdated :)